uiux-challenges/habit-tracker-ui/Infinite-L00pBaCk/style.css

```
@@ -0,0 +1,461 @@
+
+.nav-fab:hover {
+  transform: translateY(-14px) scale(1.05);
+  box-shadow: 0 12px 36px rgba(124, 92, 252, 0.5);
+}
+
+.nav-fab:active {
+  transform: translateY(-10px) scale(0.95);
+}
+
+/* ============ ANIMATIONS ============ */
+@keyframes fadeInUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)